debugging-contd/problem-set-debugging-contd.md

```diff
@@ -230,72 +191,108 @@ Some examples of things to try:
 - Grab the code and run it inside of VSCode with the debugger.
 
 ##### !end-explanation
-
 ### !end-challenge
-
 <!-- prettier-ignore-end -->
-<!--END CHALLENGE-->
 
-<!--BEGIN CHALLENGE-->
 <!-- prettier-ignore-start -->
-
 ### !challenge
-
 * type: paragraph
 * id: 882c916d-55ab-4c53-888b-8b8d68ea8021
 * title: Debugging Continued
-
 ##### !question
 
-Why doesn't Moira's `one_generation` function work? Why are all of the cells dead in the second generation in the test?
+Why doesn't Dahlia's `encode_strand` function work? What is happening that means the last base of the genetic sequence never gets added?
 
 ##### !end-question
-
-<!--optional-->
 ##### !explanation
 
-Conway's Game of Life calculates the alive or dead status of each cell based on the state of the previous generation. Moira's code is *changing* the state of the cells before the code finishes the calculations for each cell.
-
-<br/>
+The function `encode_strand` that is compressing the sequence is not accounting for adding the last base to the compressed string. 
```

Comment:
We need a break tag between line 209 and 210 to visually separate the two paragraphs.